src/chainstate/stacks/miner.rs

```diff
@@ -292,21 +304,37 @@ impl<'a> StacksMicroblockBuilder<'a> {
     }
 
     /// Mine the next transaction into a microblock.
-    /// Returns true/false if the transaction was/was not mined into this microblock.
+    ///
+    /// This is a wrapper around `StacksChainState::process_transaction` that also checks certain
```

Comment:
"Wrapper" is not really the right word here.  The `mine_next_transaction` method creates a block by processing a sequence of transactions it pulls from the mempool.  The inner call to `StacksChainState::process_transaction` is used to faithfully create the MARF trie for this block and calculate the MARF state root for the block header.  This is not the same as being a wrapper, since `mine_next_transaction` is not meant to be used to process blocks.